.changeset/calm-apes-collect.md

```diff
@@ -0,0 +1,7 @@
+---
+'@firebase/webchannel-wrapper': minor
+---
+
+Upgrade to the latest version of google closure library and compiler. This dependency will be
```

Comment:
Google stylizes this as "Closure Library", so this should be "Google Closure Library and Compiler": https://developers.google.com/closure/library

Reply:
Done